wordpress-mobile/WordPress-iOS
# Apply post styles into webview

Fixes #14210

(Also address #13862, #9525 and #13864)

This PR renders the post content into a webview applying the correct styles.

The styles were extracted from Calypso with a few modifications.

Noticed that header and footer are going to be tackled in the following task.

### To test

(To render the post into a webview remember to enable the feature flag in app settings)

My recommendation to test this PR is to keep one device (or simulator) with the current in-production Reader and the other with this branch. Then, open a post on both and compare — the post should appear **similar** on both. In this phase I'm looking for very outstanding bugs (instead of fixing everything).

Besides that, please:

1. Test with iPhone and iPad (note that in iPad we're using readable margins)
2. Test light and dark mode
3. Test portrait and landscape

PR submission checklist:
